akka-stream/src/main/scala/akka/stream/IOResult.scala

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (C) 2016 Typesafe Inc. <http://www.typesafe.com>
  */
-package akka.stream.io
+package akka.stream
```

Comment:
Then I don't get it, it is about IO so would have expected it to live in the IO package hm...
*:class:`IOResult` is a type that IO operations return in Akka Streams*

I don't mind enough to battle over it but it seems like a weird move if only for the nicer import hm